winml/lib/Api/LearningModelSession.cpp

```diff
@@ -255,13 +221,31 @@ LearningModelSession::Run(
   auto device = device_.as<LearningModelDevice>();
   CWinMLAutoLock lock(!device->IsCpuDevice() ? &evaluate_lock_ : nullptr);
   // TODO : set the run_options
-  onnxruntime::RunOptions run_options;
+  Ort::RunOptions run_options;
+  binding_impl->BindUnboundOutputs();
 
-  com_ptr<_winmla::IIOBinding> io_binding;
-  io_binding.attach(GetIOBinding(binding_impl, model_));
+  std::vector<const char*> inputNames_c;
```

Comment:
>inputNames_c [](start = 27, length = 12)

why can't we just use binding_impl->GetInputNames directly?

Reply:
The C++ API needs const char* const * to represent a vector of std::string